### ISSUE TITLE
Translation Missing: Turkish

### ISSUE THREAD

**Issue by noche-x** [mrneo240/psp-homebrew-store#3]
**Which Language does this translation cover**
Turkish.

**Additional context**
no.

**Note!**
If unsure where a particular string is used, either browse the store: [PSP Homebrew Store](https://psp-dev.org/hb/) or ask here for clarification.

Original English Translation:
```ini
; Base template file, Also sets default fallbacks
MSG_PSP_WELCOME = %s'a Hoşgeldin!
MSG_HOMEBREW_INTRO = Bu Homebrew Mağazası archive.org'tan taze ve canlı içerik sunar özellikle Sony PSP'ler için, bilgisayar lazım değildir.<br /><br />Bu site şu anda PSP Homebrew Comunity Discord'unda süren hombrew koruma eforu gösterenler sayesinde mümkün olmuştur.

MSG_PSP_CHANNEL = %s için Yüklenebilir Kanal
MSG_CHANNEL_INTRO = İkona tıklayarak bu mağazayı PSP'nize direk olarak indirebilirsiniz!<br />  XMB'deki İnternet Radyosu bölümüne kısayol indirecektir!<br /><br />Bu ikon PSP Homebrew Community Discord'undandır.<br />Discord linki: <a href="https://discord.gg/bePrj9W">https://discord.gg/bePrj9W</a>

MSG_PSP_HOME = Ana Sayfa
MSG_PSP_GAMES = Oyunlar
MSG_PSP_APPS = Uygulamalar
MSG_PSP_PORTS = Portlar
MSG_PSP_EMULATORS = Emülatörler
MSG_PSP_DEMOS = Demolar
MSG_PSP_SEARCH = Ara
MSG_PSP_POPULAR = Popüler
MSG_PSP_ABOUT = Hakkında

MSG_TABLE_TITLE = Başlık
MSG_TABLE_DESCRIPTION = Açıklama
MSG_TABLE_SIZE = Boyut
MSG_TABLE_DOWNLOAD = İND#

MSG_AUTHOR = Yapımcı
MSG_DATE = Çıkş tarihi
MSG_VERSION = Versiyon

MSG_SIZE_NOTICE_PHAT = Yükleyici PSP1000 namı diğer Phat'te hata verebilir
MSG_SIZE_NOTICE_SLIM = Yükleyici her PSP modellerinde hata verebilir!

MSG_DOWNLOAD = Yükle!
MSG_FURTHER_INFO = Daha Fazla Bilgi (Sadece Masaüstü Tarayıcılar için)
MSG_REVIEWS = Yorumlar
MSG_ON = 'da

MSG_SEARCH = %s'i Ara
MSG_SEARCH_GO = Git
MSG_SEARCH_NOTE = <b>Not:</b> Sadece isimler arasında arar.
MSG_SEARCH_RESULTS = Arama Sonuçları

MSG_POPULAR = En Popüler

MSG_ABOUT = Hakkında
MSG_NOTICE = Duyuru
MSG_NOTICE_TEXT = Bu <b>bedava</b> yazılımdır!

MSG_PAGE = Sayfa %d
```

**Comment by mrneo240** [comment]
`MSG_FURTHER_INFO = Daha Fazla Bilgi (Sadece Masaüstü Tarayıcılar için)`
this line in english moving forward will read 
`MSG_FURTHER_INFO = Further Information (Desktop Only)`

sorry this change was just decided, please make the necessary changes

**Comment by noche-x** [comment]
yeah just
`MSG_FURTHER_INFO = Daha Fazla Bilgi (Sadece Masaüstü için)`

**Comment by mrneo240** [comment]
implemented by 1e382d154fe43985a33078bab309a51e8f629885
